drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -161,7 +161,7 @@ function DragAndDropEditBlock(runtime, element, params) {
                             $(this).addClass('hidden');
                             $('.save-button', element).parent()
                                 .removeClass('hidden')
-                                .one('click', function submitForm(e) {
+                                .on('click', function submitForm(e) {
```

Comment:
@e-kolpakov OK. But now that I'm looking at the surrounding code I'm wondering what the purpose of [this code](https://github.com/open-craft/xblock-drag-and-drop-v2/blob/c1941da6629d38d41b5b2cc8b162b323dc5f80d6/drag_and_drop_v2/public/js/drag_and_drop_edit.js#L168-L171) is. It looks like it was supposed to work around the behavior you're describing. If that's the case, we should probably remove it -- the `.on` solution seems cleaner anyway.